### ISSUE TITLE
Fetch Community from Logged In User

### ISSUE THREAD

**Issue by Dinesh846** [hserv/coordinated-entry#18]
Fetch community from logged in user and pass that to survey and inventory micro services to get response for a particular community.

**Comment by eric-jahn** [comment]
Waiting on Issue #137 in HMISLynk for this to be resolveable: https://github.com/servinglynk/hmis-lynk-open-source-docs/issues/137

**Comment by eric-jahn** [comment]
Duplicate of#10
